yt/visualization/plot_window.py

```diff
@@ -372,6 +375,26 @@ def _recreate_frb(self):
         for key in self.override_fields:
             self._frb[key]
 
+    @property
+    def _swap_axes(self):
```

Comment:
I think the following method could be simply named `swap_axes` instead of `toggle_swap_axes`.
How about naming this property `_has_swapped_axes` to avoid ambiguity and clarify it's a boolean ? this name doesn't need to be concise since it's not public API.
```suggestion
    def _has_swapped_axes(self) -> bool:
```
Of course this suggestion isn't self contained, you'd have to adapt the rest of the code too

Reply:
yup, good suggestion. I went with `toggle_` because we had already had a `toggle_right_handed` method. But grammatically I like just having `swap_axes`. I'll change this.

Reply:
Oh. I had no idea we had a `toggle_right_handed` method. It looks exactly identical to what I'd expect from the `flip_horizontal_axis` method I suggested already, and it shows the way for how to go about a `flip_vertical_axis` method ! So I'm going to go with a bold proposal here and suggest we alias `toggle_right_handed` to `flip_horizontal_axis` and deprecate the original method because it represents only _one_ of the two possible implementations for its name.

Reply:
latest batch should resolve this!